spec/controller/post_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get :index, params: { user_id: 1 }
+      expect(response).to be_successful
+    end
+
+    it 'renders the index view' do
+      get :index, params: { user_id: 1 }
+      assert 'title', 'Here is a list of posts for User Id: 1'
+    end
```

Comment:
- Good job done so far, but according to the project requirements, you are to write tests that ensure that the correct template is rendered, just as you did in the `post#show` test below. Please fix this 😄